packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -91,6 +128,7 @@ export function Autocomplete({
 
   function handleInputChange(newText: string) {
     updateInput(newText);
+    onChange({ label: newText });
```

Comment:
I don't think we want to call this here as it will fire constantly as they type regardless of a valid option being chosen.